easybuild/easyconfigs/p/Python/Python-2.7.14-iomkl-2018a.eb

```diff
@@ -0,0 +1,175 @@
+name = 'Python'
+version = '2.7.14'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'iomkl', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['304c9b202ea6fbd0a4a8e0ad3733715fbd4749f2204a9173a58ec53c32ea73e8']
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.11'),
+    ('libreadline', '7.0'),
+    ('ncurses', '6.0'),
+    ('SQLite', '3.22.0'),
```

Comment:
I probably suggested `3.22.0` myself here, but we should stick to `3.21.0` like in the `intel/2018a` easyconfig, for which an easyconfig already exists, to avoid introducing conflicts between different packages that depend on `SQLite` going forward.

Also remove `SQLite-3.22.0-GCCcore-6.4.0.eb` from this PR.

I need to look into an automated check for this, but I didn't get around to it yet...